portal-impl/src/com/liferay/portal/service/permission/LayoutPermissionImpl.java

```diff
@@ -244,11 +245,15 @@ public boolean containsWithoutViewableGroup(
 			// action
 
 			long parentLayoutId = layout.getParentLayoutId();
+			long layoutGroupId = layout.getGroupId();
```

Comment:
This resolves an exception that occurs when navigating to a child page within user personal pages if that child page comes from a site template attached to a user group.